__tests__/app.test.js

```diff
@@ -19,8 +19,9 @@ describe('GET /api/topics', () => {
         .get("/api/topics")
         .expect(200)
         .then(({body})=> {
-            expect(Array.isArray(body)).toBe(true);
-            body.forEach((topic) => {
```

Comment:
At the moment, this will pass until it finds a topic which doesn't have the specified keys - which is good - although what if we get an empty array back? It won't fail, because it won't find any topics that _don't_ have those properties, but nor should it pass. Also checking the length of the returned array here will give your testing far more integrity.